docs/topics/native/native-memory-manager.md

```diff
@@ -62,10 +66,27 @@ To track GC-related pauses in your app:
 
    Here, each blue blob on the lowest graph represents a separate signpost event, which is a GC pause.
 
+### Optimize GC performance
+
+To improve GC performance, you can try a couple of options aimed at decreasing the GC pause time:  
+
+* For the marking phase, you can use concurrent marking. It allows to run the marking phase of the garbage collection
+  simultaneously with application threads.
+
+  The feature is currently [Experimental](components-stability.md#stability-levels-explained). To enable it, set the
+  following option in your `gradle.properties` file:
+  
+  ```properties
+  kotlin.native.binary.gc=cms
+  ```
+
+* For the next phase of handling weak references, you can enable concurrent processing by using the
+  `-Xbinary=concurrentWeakSweep=true` compiler option.
```

Comment:
BTW, we need to choose a single idiomatic way of setting binary options and stay with it. Here we suggest to use `-Xbinary=...` command line argument, while a few likes above we do it through a `kotlin.native.binary...` property.

Reply:
True, I changed all of them to `kotlin.native` properties, please check if I got it right

Reply:
Thank you. Unfortunately, this trick works only with the `-Xbinary=...` options. All the other such as `-Xruntime-logs=` and `-Xallocator=` have to remain in the command line form.